03/app.test.js

```diff
@@ -1 +1,31 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('throw exception when min is not a number', () => {
+    function callRandomNumber() {
+        randomNumber('3', 4)
+    }
+        expect(callRandomNumber).toThrow()
+})
+
+it('throw exception when max is not a number', () => {
+    function callRandomNumber() {
+        randomNumber(3, '4')
+    }
+        expect(callRandomNumber).toThrow()
+})
+
+it('throw exception when max <= min', () => {
+    function callRandomNumber() {
+        randomNumber(4, 3)
+        randomNumber(4, 4)
```

Comment:
Raczej bym nie sprawdzał obu przypadków.
Jeśli 1 zawsze będzie prawidłowo działał to nigdy nie sprawdzisz czy 2 działa prawidłowo ;)